openslides/motions/config_variables.py

```diff
@@ -39,6 +39,15 @@ def get_config_variables():
         subgroup='General')
 
     yield ConfigVariable(
+        name='motions_recommendations_by',
+        default_value='Recommendation committee',
+        label='Name of recommendation committee',
+        weight=312,
+        group='Motions',
+        subgroup='General',
+        translatable=True)
```

Comment:
Move the config to the end of the general subgroup. 
How can we deactivate the Recommendation field in template? Maybe we need a enable config like for amendments? (because not all users want to use/see this special recommendation feature)